src/modules/advancedOrders/state/advancedOrdersSettingsAtom.ts

```diff
@@ -0,0 +1,26 @@
+import { atom } from 'jotai'
+import { atomWithStorage } from 'jotai/utils'
+
+export interface AdvancedOrdersSettingsState {
+  readonly showRecipient: boolean
+}
+
+export const defaultAdvancedOrdersSettings: AdvancedOrdersSettingsState = {
+  showRecipient: false,
+}
+
+export const advancedOrdersSettingsAtom = atomWithStorage<AdvancedOrdersSettingsState>(
+  'advanced-orders-settings-atom:v2',
```

Comment:
Can you start with `v0` instead of `v2`?

Reply:
Good catch!